pkg/apis/folder/v0alpha1/types.go

```diff
@@ -37,7 +37,8 @@ type FolderInfoList struct {
 
 	// +listType=map
 	// +listMapKey=uid
-	Items []FolderInfo `json:"items,omitempty"`
+	// TODO: Make a new version with `items` => `infoItems`. I just did this to work around it in dev.
```

Comment:
fixed in:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.32.md

with https://github.com/kubernetes/kubernetes/pull/126809